python/hsfs/embedding.py

```diff
@@ -33,6 +33,17 @@ class SimilarityFunctionType:
 
 @dataclass
 class EmbeddingFeature:
+    """
+    Represents an embedding feature.
+
+    # Arguments
+        - name: The name of the embedding feature.
+        - dimension: The dimensionality of the embedding feature.
+        - similarity_function_type: The type of similarity function used for the embedding feature
+          (default is SimilarityFunctionType.L2).
+        - feature_group: The feature group associated with the embedding feature.
+        - embedding_index: The embedding index to which the feature belongs.
```

Comment:
The name of the index in the vector database that will store the embedding feature.